<launch>
  <node name="gnss_parser" pkg="gnss_parser" type="gnss_nmea_parser.py" output="screen">
    <param name="serial_port" value="/dev/ttyUSB1" />
    <param name="baud_rate" value="115200" />
  </node>
</launch>
